src/main/java/net/sf/jabref/gui/dbproperties/DatabasePropertiesDialog.java

```diff
@@ -105,10 +105,11 @@ private void init(JFrame parent) {
         browseFile.addActionListener(BrowseAction.buildForDir(parent, fileDir));
         browseFileIndv.addActionListener(BrowseAction.buildForDir(parent, fileDirIndv));
 
-        setupSortOrderConfiguration();
 
-        FormBuilder builder = FormBuilder.create().layout(new FormLayout("left:pref, 4dlu, left:pref, 4dlu, pref:grow",
-                "pref, 2dlu, pref, 2dlu, pref, 2dlu, pref, 2dlu, pref, 2dlu, pref, 2dlu, pref, 2dlu, pref, 2dlu, pref, 2dlu, pref, 2dlu, pref, 2dlu, pref, 2dlu, pref, 2dlu, pref, 2dlu, pref, 140dlu, pref,"));
+        setupSortOrderConfiguration();
+        FormLayout form = new FormLayout("left:pref, 4dlu, left:pref, 4dlu, pref:grow",
```

Comment:
Change the grow to the third column? Now, the browse button is resized. better if the text field is resized.